lib/timers.js

```diff
@@ -296,6 +296,9 @@ exports.enroll = function(item, msecs) {
 
   // Ensure that msecs fits into signed int32
   if (msecs > TIMEOUT_MAX) {
+    process.emitWarning(`Timer duration overflow: ${msecs} does not fit into ` +
```

Comment:
This can be simplified to:

```js
process.emitWarning(`${msecs} does not fit into a 32-bit signed integer.\n` +
                                   `Timeout duration was truncated to ${TIMEOUT_MAX}.`,
                                   'TimerOverflowWarning');
```

The `TimerOverflowWarning` name would be printed to the console and gives a consistent label to key off.